proto/tendermint/abci/types.proto

```diff
@@ -212,6 +211,12 @@ message ResponseCheckTx {
   repeated Event events     = 7
       [(gogoproto.nullable) = false, (gogoproto.jsontag) = "events,omitempty"];
   string codespace = 8;
+  string sender    = 9;
```

Comment:
Is the SDK using this?

Reply:
No, but this is a drop-in placement of what 0.35.x has.